src/unstract/adapters/x2text/llama_parse/src/llama_parse.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import os
+from typing import Any, Optional
+
+from llama_parse import LlamaParse 
+
+from unstract.adapters.exceptions import AdapterError
+from unstract.adapters.x2text.llama_parse.src.constants import LlamaParseConfig
+from unstract.adapters.x2text.x2text_adapter import X2TextAdapter
+from httpx import ConnectError
```

Comment:
Do check if this is present in the std library of python - if not, do add an entry to the root `pyproject.toml`

Reply:
httpx is already present as a transitive dependency in the project.

Reply:
Since we directly import it - we should mention it as a dependency in our project as well. If `llama-parse` chooses to remove this lib for some reason, our code would break when we do update the deps